src/services/tokenService.js

```diff
@@ -0,0 +1,28 @@
+const Token = require('../models/token.js');
+
+const save = async (userId, newToken) => {
+  const token = await Token.findOne({ where: { userId } });
+
+  if (!token) {
+    await Token.create({ userId, refreshToken: newToken });
+
+    return;
+  }
+
+  token.refreshToken = newToken;
+  await token.save();
+};
+
+const getByToken = (refreshToken) => {
+  return Token.findOne({ where: { refreshToken } });
+};
+
+const remove = (userId) => {
+  return Token.destroy({ where: { userId } });
```

Comment:
Consider adding error handling for the database operation to catch and log any errors that occur during the token removal process. This will help in diagnosing issues if the operation fails.